src/routes/v0/health.ts

```diff
@@ -0,0 +1,17 @@
+const { Router } = require('express');
+const models = require('../../models');
+
+const healthRouter = Router();
+
+import { SerializedResponse } from '../../types';
```

Comment:
I see your using `import` for getting `SerializedResponse`.
Could we also fetch `Router` using `import`? e.g.: `import { Router } from 'express';`

Reply:
You're right! I think we can replace that one, and also the rest of `requires` spread in the application.
It seems that is better to use `import` instead of `require`:
https://www.educba.com/require-vs-import/